src/SUMMARY.md

```diff
@@ -145,6 +145,7 @@
     - [Debugging LLVM](./backend/debugging.md)
     - [Backend Agnostic Codegen](./backend/backend-agnostic.md)
     - [Implicit Caller Location](./backend/implicit-caller-location.md)
+- [Libraries and Metadata](./backend/libs-and-metadata.md)
```

Comment:
Perhaps instead put this under "High-level compiler architecture"? That's where queries and serialization are discussed, which are somewhat related.